### PR TITLE
Cascade design refers to symmetric ideal cascade

### PR DESCRIPTION
**Pull request by bam241**
The Villani's book refers to them as `ideal symmetric cascade`so I kept the same order... (symmetric ideal cascade sounded better to me at first...)

I also included the new production graph

### CODE REVIEW

File: abstract.tex [hunk 1]
```diff
@@ -7,7 +7,7 @@
 The goal of this work focuses on developing the necessary understanding of
 potential misuse of enrichment cascades, contributing to more effective
 international safeguards designs and approaches.  While it is straightforward to
-design an enrichment cascade under ideal conditions as a function of the
+design an symmetric enrichment cascade under ideal conditions as a function of the
```
Comment (gonuke) [inline abstract.tex]: "a symmetric enrichment"  instead of "an"

File: motivation.tex [hunk 2]
```diff
@@ -6,9 +6,9 @@ \section{Motivation}
 enrichment while rejecting tails at a fixed assay.
 
 With the current international tensions regarding enrichment capabilities, this
-work aims to measure the effectiveness of an enrichment cascade when used outside of
-its designed scope and quantify the attractiveness of such way to build up
-significant quantity of \gls{HEU}.
+work aims to measure the effectiveness of an symmetric enrichment cascade when
```
Comment (gonuke) [inline motivation.tex]: "a symmetric cascade" instead of "an"

File: theory.tex [hunk 3]
```diff
@@ -25,7 +25,7 @@ \subsection{Centrifuge properties}
 
 \subsection{Cascade Design}
 
-The cascade is built as an ideal cascade, with no losses in the separative work,
+The cascade is built as a ideal symmetric cascade, with no losses in the separative work,
```
Comment (gonuke) [inline theory.tex]: "an ideal" instead of "a"

**Review by gonuke — CHANGES_REQUESTED** [review]
some article agreement problems

**Comment by bam241** [comment]
fixed!

**Review by gonuke — APPROVED** [review]

**Pull request merged**